Description: Remove broken FvwmCommand.{pm,sh} scripts from build.
  The FvwmCommand.pm and FvwmCommand.sh wrapper scripts are broken
  and don't generate correctly. Remove them from the build.
  Remove wrapper info from manual page.
Author: Jaimos Skriletz <jaimosskriletz@gmail.com>
Forwarded: https://github.com/fvwmorg/fvwm/pull/97/commits/9be57d08a7eff85c1b5c83ef9639a625ab85b646
Last-Update: 2022-10-09
--- a/modules/FvwmCommand/Makefile.am
+++ b/modules/FvwmCommand/Makefile.am
@@ -6,29 +6,14 @@ SUBDIRS = scripts
 
 bin_PROGRAMS = FvwmCommand
 moduledir = @FVWM_MODULEDIR@
-module_DATA = FvwmCommand.sh FvwmCommand.pm
 
 FvwmCommand_SOURCES = FvwmCommand.c FvwmCommand.h fifos.c
 
-FvwmCommand.sh: findcmd.pl $(top_srcdir)/fvwm/functable.c
-	if test -n "$(PERL)" -a -x "$(PERL)"; then \
-		$(PERL) $(srcdir)/findcmd.pl $(bindir) -sh \
-		< $(top_srcdir)/fvwm/functable.c > $@; \
-	else echo '# Did not find perl during fvwm install' > $@; fi
-
-FvwmCommand.pm: findcmd.pl $(top_srcdir)/fvwm/functable.c
-	if test -n "$(PERL)" -a -x "$(PERL)"; then \
-		$(PERL) $(srcdir)/findcmd.pl $(bindir) \
-		< $(top_srcdir)/fvwm/functable.c > $@; \
-	else echo '# Did not find perl during fvwm install' > $@; fi
-
-CLEANFILES = FvwmCommand.sh FvwmCommand.pm
-
 FvwmCommand_DEPENDENCIES = $(top_builddir)/libs/libfvwm.a
 
 man_MANS = FvwmCommand.1
 
-EXTRA_DIST = $(man_MANS) findcmd.pl
+EXTRA_DIST = $(man_MANS)
 
 LDADD = -L$(top_builddir)/libs -lfvwm
 
--- a/modules/FvwmCommand/FvwmCommand.1.in
+++ b/modules/FvwmCommand/FvwmCommand.1.in
@@ -281,33 +281,6 @@ FvwmCommand exits if no information or e
 time period unless option -m is used.
 The default is 500 ms. This option overrides this default value.
 
-.SH WRAPPER
-.sp
-.sp
-FvwmCommand.sh has bourne shell function definitions
-to keep the syntax similar to fvwm configuration file.
-This file is to be sourced:
-.EX
-\&. FvwmCommand.sh
-.br
-DesktopSize 5x5
-.EE
-.br
-FvwmCommand.pm is for perl in order
-to keep the syntax similar to fvwm configuration file.
-Commas can be used to separate fvwm commands' arguments.
-.EX
-use FvwmCommand;
-if( $ARGV[0] eq 'home' ) {
-    Desk 0,0; GotoPage '1 1';
-}elsif( $ARGV[0] eq 'jump' ) {
-    Desk "0 2"; GotoPage 0, 1;
-}
-.EE
-Although arguments in FvwmCommand are not case sensitive as fvwm,
-the functions defined in FvwmCommand.sh and FvwmCommand.pl are case sensitive.
-
-
 .SH ERRORS
 If the following error message show up, it is most likely that FvwmCommandS
 is not running.
